src/MergePlugin.php

```diff
@@ -368,6 +369,36 @@ function(&$path) use ($packagePath) {
     }
 
     /**
+     * @param RootPackage $root
+     * @param CompletePackage $package
+     * @param string $path
+     */
+    protected function mergeDevAutoload(
+        RootPackage $root,
+        CompletePackage $package,
+        $path
+    ) {
+        $autoload = $package->getDevAutoload();
```

Comment:
Looks like you are duplicating quite some code from the `mergeAutoload` function. How about factoring out the common part?